### DISCUSSION TITLE
Anyone encountered data not properly saving in UI? [v0.90.5]

### DISCUSSION THREAD

**Discussion by RawmanGrey** [nocodb/nocodb#1812] [Q&A]
While I was playing around NocoDB UI testing cases for the issue, quite unpleasant thing happened: data I was entering was not always properly saved and was disappearing on page updates. This issue I was able to reproduce running test docker setup with `docker run -p 8080:8080 -d nocodb/nocodb` command. So the question is, am I alone with this?) 

<details> 
  <summary>recorded bug as .gif</summary>

![Peek 2022-04-21 12-43](https://user-images.githubusercontent.com/61871501/164430926-51c167f7-6718-405a-89e4-7cf77dbc778e.gif)
</details>

**Comment by o1lab** [comment]
Created an issue of this. Looking into this.
